emacs/merlin.el

```diff
@@ -324,16 +320,14 @@ The association list can contain the following optional keys:
     (message "Last commands executed, most recent at the end:\n%s"
              (mapconcat 'identity buf "\n"))))
 
-(defun merlin/buffer-substring (start end)
-   "Return content of buffer between two points or empty string if points are not valid"
-   (if (< start end) (buffer-substring-no-properties start end) ""))
+(defun merlin-buffer-substring (start end)
+  "Return content of buffer between two points or empty string if points are not valid"
+  (if (< start end) (buffer-substring-no-properties start end) ""))
 
-(defun merlin-lookup (key list &optional default)
+(defsubst merlin-lookup (key list &optional default)
```

Comment:
Why `defsubst`?

I don't see a point to this function anymore. You can just inline it everywhere.

Reply:
Sure, but each call site would need to specify the `nil` value for the TEST arg. I can do this if you like but it didn't seem like a big win when I considered it.

An alternate modern way to handle this would be to use `let-alist` and then `or` to apply defaults btw.

Reply:
Thanks for taking a look!

Reply:
I'll let you decide what's the best way to handle this. Using `let-alist` seems fine to me.